internal/migrations/20201019194303_change_overrides_to_text.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/openshift/assisted-service/models"
+	gormigrate "gopkg.in/gormigrate.v1"
+)
+
+const id = "20201019194303"
+
+func changeOverridesToText() *gormigrate.Migration {
```

Comment:
Consider checking the column type and if migration is required, so we can just skip it and won't fail in case it's not required.

Reply:
I don't think it will fail.

This is an interesting case because I'm also changing the swagger so a new deployment would have the correct type but a redeploy with the new code would need the migration.

Do you think we should not put changes that need migrations into the swagger or would that be too hard to keep track of?

Reply:
Yup, just verified that a new deploy (where the automigration would create the type correctly) doesn't fail on this.
Additionally the migration is still recorded in the `migrations` table in this case.